src/github-patcher.ts

```diff
@@ -65,33 +68,83 @@ const hideHighlights = (): void => {
 
   const highlightsComponent = highlightsH2Node?.parentElement;
   if (highlightsComponent) {
-    hide(highlightsComponent);
+    if (shouldHide) {
+      hide(highlightsComponent);
+    } else {
+      show(highlightsComponent);
+    }
+  }
+};
+
+let styleElement: HTMLStyleElement | null = null;
+
+const injectHideCSS = (): void => {
+  if (styleElement) {
+    styleElement.remove();
+  }
+  styleElement = document.createElement('style');
+  styleElement.textContent = HIDE_CSS_CONTENT;
+  styleElement.id = 'depop-styles';
+  document.head.appendChild(styleElement);
+};
+
+const injectShowCSS = (): void => {
+  if (styleElement) {
+    styleElement.remove();
+  }
+  styleElement = document.createElement('style');
+  styleElement.textContent = SHOW_CSS_CONTENT;
+  styleElement.id = 'depop-styles';
+  document.head.appendChild(styleElement);
+};
```

Comment:
It shows multiple Achievements

<img width="514" height="541" alt="Image" src="https://github.com/user-attachments/assets/242e016c-8f28-4831-9b72-151d0a815cab" />